smriprep/workflows/anatomical.py

```diff
@@ -258,12 +270,12 @@ def init_anat_preproc_wf(skull_strip_template, fs_spaces, template, debug,
                 ('subjects_dir', 'inputnode.subjects_dir'),
                 ('subject_id', 'inputnode.subject_id')]),
             (anat_template_wf, surface_recon_wf, [('outputnode.t1_template', 'inputnode.t1w')]),
-            (skullstrip_ants_wf, surface_recon_wf, [
-                ('outputnode.out_file', 'inputnode.skullstripped_t1'),
-                ('outputnode.out_segs', 'inputnode.ants_segs'),
-                ('outputnode.bias_corrected', 'inputnode.corrected_t1')]),
-            (skullstrip_ants_wf, applyrefined, [
-                ('outputnode.bias_corrected', 'in_file')]),
+            (brain_extraction_wf, surface_recon_wf, [
+                (('outputnode.bias_corrected', _pop), 'inputnode.skullstripped_t1'),
```

Comment:
Bias corrected isn't the same as skull stripped, is it?

Reply:
Yes, it was. This is one of the reasons why I wanted to translate this workflow into nipype. In a future PR I want to address these:

- Bias-corrected should not be the same as skull stripped
- Add an additional N4 pass on the original image (not truncated, as it is now) with the brain mask, so both bias-corrected and skull-stripped have this more accurate correction.
- Reuse the initial transforms in the normalization workflow if the template used is the same (and consequently move to MNI152NLin2009cAsym as target template for skull-stripping).

Reply:
But we save the bias-corrected file as the t1_preproc (L253), and that's not skull-stripped.

Reply:
You are right. It should be fixed now.